cabal.project

```diff
@@ -114,30 +114,61 @@ constraints:
   , typed-protocols >= 0.1.0.1
   , typed-protocols-examples >= 0.1.0.1
 
-  , cardano-ledger-alonzo ^>= 0.1.1.1
-  , cardano-ledger-alonzo-test ^>= 0.1.1.1
-  , cardano-ledger-babbage ^>= 0.1.1.1
-  , cardano-ledger-babbage-test ^>= 0.1.1.1
-  , byron-spec-chain ^>= 0.1.1.2
-  , cardano-crypto-wrapper ^>= 1.4.1
-  , cardano-crypto-test ^>= 1.4.2
-  , byron-spec-ledger ^>= 0.1.1.2
-  , cardano-ledger-byron ^>= 0.1.1.1
-  , cardano-ledger-byron-test ^>= 1.4.1
-  , cardano-ledger-conway ^>= 0.1.1.1
-  , cardano-ledger-shelley-ma ^>= 0.1.1.1
-  , cardano-ledger-shelley-ma-test ^>= 0.1.1.1
-  , cardano-ledger-shelley ^>= 0.1.1.1
-  , cardano-ledger-shelley-test ^>= 0.1.1.1
-  , cardano-data ^>= 0.1.1.1
-  , cardano-ledger-core ^>= 0.1.1.1
-  , cardano-ledger-pretty ^>= 0.1.1.1
-  , cardano-protocol-tpraos ^>= 0.1.1.1
-  , non-integral ^>= 0.1.1.1
-  , set-algebra ^>= 0.1.1.1
-  , small-steps ^>= 0.1.1.1
-  , small-steps-test ^>= 0.1.1.1
-  , vector-map ^>= 0.1.1.1
 
 allow-newer:
     Unique:hashable
+
+
+source-repository-package
```

Comment:
This stuff will disappear in favor of manual bounds, after packages from base and ledger have bee released to CHaPs